packages/end-to-end-testing/lib/chains.ts

```diff
@@ -0,0 +1,243 @@
+import { getAddress, isNetworkName, ZetaNetworkName } from "@zetachain/addresses";
+import { isLocalNetworkName, NetworkName } from "@zetachain/addresses/src/addresses.helpers";
```

Comment:
```suggestion
import { isLocalNetworkName, NetworkName } from "@zetachain/addresses/addresses.helpers";
```

```suggestion
import { isLocalNetworkName, NetworkName } from "@zetachain/addresses";
```

Maybe one of those should work? If the second doesn't we can re-export from an index.ts file on the addresses package.

Reply:
I had to re-export from an index.ts to make this work.